Reference/conformance/conformance.pb.swift

```diff
@@ -1,10 +1,10 @@
-/*
- * DO NOT EDIT.
- *
- * Generated by the protocol buffer compiler.
- * Source: conformance/conformance.proto
- *
- */
+// DO NOT EDIT.
+//
+// Generated by the Swift generator plugin for the protocol buffer compiler.
+// Source: conformance/conformance.proto
+//
+// For information on using the generated types, please see the documenation:
+//   https://github.com/apple/swift-protobuf/blob/master/Documentation/API.md
```

Comment:
Minor quibble:  I would just refer to https://github.com/apple/swift-protobuf/ here and let the README point people to more specific documentation.  In particular, these generated files get checked into source control and can live there for years before they're regenerated.